### PR TITLE
Add missing error assignments

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

File: plugins/router/template/router.go [hunk 1]
```diff
@@ -63,11 +63,11 @@ func (r *templateRouter) Commit() error {
 		return err
 	}
 
-	if r.writeConfig(); err != nil {
+	if err = r.writeConfig(); err != nil {
```
Comment (smarterclayton) [inline plugins/router/template/router.go]: Can you make these `:=`, just to be even clearer?  Thanks!

Comment (kargakis) [inline plugins/router/template/router.go]: @smarterclayton done.

**Comment by mfojtik** [comment]
LGTM

**Comment by bparees** [comment]
lgtm [merge]

**Pull request merged**